### PR TITLE
Code Commit

### PR DESCRIPTION
**Pull request by KFloodBCGov**
Initial commit of the code base for the PCSS webMethods modernization project

### CODE REVIEW

File: src/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
```
Comment (alexjoybc) [inline src/Dockerfile:1]: you can take this file off for now, and it has DPS ref in there

File: src/openshift.dockerfile [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#############################################################################################
```
Comment (alexjoybc) [inline src/openshift.dockerfile:1]: same here

File: src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl.renamed [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+                <soap12:body parts="parameters" use="literal"/>
+            </wsdl:output>
+        </wsdl:operation>
+    </wsdl:binding>
+    <wsdl:service name="pcssReport">
+        <wsdl:port name="JusticePCSSCommon_wsProvider_pcssReport_Port" binding="tns:JusticePCSSCommon_wsProvider_pcssReport_Binder">
+            <soap12:address location="http://pcssreports.pcss.open.gov.bc.ca:5555/ws/JusticePCSSCommon.wsProvider:pcssReport/JusticePCSSCommon_wsProvider_pcssReport_Port"/>
+        </wsdl:port>
+    </wsdl:service>
+</wsdl:definitions>
```
Comment (akroon3r) [inline src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl.renamed:188]: missing new line

File: src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+                <soap12:body parts="parameters" use="literal"/>
+            </wsdl:output>
+        </wsdl:operation>
+    </wsdl:binding>
+    <wsdl:service name="pcssReport">
+        <wsdl:port name="JusticePCSSCommon_wsProvider_pcssReport_Port" binding="tns:JusticePCSSCommon_wsProvider_pcssReport_Binder">
+            <soap12:address location="http://reeks.bcgov:5555/ws/JusticePCSSCommon.wsProvider:pcssReport/JusticePCSSCommon_wsProvider_pcssReport_Port"/>
+        </wsdl:port>
+    </wsdl:service>
+</wsdl:definitions>
```
Comment (akroon3r) [inline src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl:176]: missing new line. also know that when I point these out its more my own OCD/PTSD. If the files are auto generated, don't worry.

File: src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl.renamed (hunk 3 above)
Comment (akroon3r) [reply src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl.renamed:188]: also know that when I point these out its more my own OCD/PTSD. If the files are auto generated, don't worry.

File: src/pcss-criminal-api/pom.xml [hunk 5]
```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.2.6.RELEASE</version>
```
[the remark this answers is not in the document]
Comment (alexjoybc) [reply src/pcss-criminal-api/pom.xml:8]: it's the specific pom for the pcss-criminal-api, it requires spring boot to be the parent

File: src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl.renamed (hunk 3 above)
Comment (alexjoybc) [reply src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl.renamed]: those are coming from webMethods directly

Comment (KFloodBCGov) [reply src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl.renamed]: Got rid of that file entirely, as I had left it in by mistake anyhow.

File: src/Dockerfile (hunk 1 above)
Comment (KFloodBCGov) [reply src/Dockerfile:1]: Done!

File: src/openshift.dockerfile (hunk 2 above)
Comment (KFloodBCGov) [reply src/openshift.dockerfile]: Done!

File: src/pcss-criminal-api/pom.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (KFloodBCGov) [reply src/pcss-criminal-api/pom.xml:8]: I... don't know?

File: src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl (hunk 4 above)
Comment (KFloodBCGov) [reply src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl:176]: The wsdl is more or less grabbed directly from the webMethods server as is.

**Pull request merged**